src/sentry/static/sentry/app/views/onboarding/projectSetup/firstEventIndicator.jsx

```diff
@@ -0,0 +1,140 @@
+import {keyframes} from 'emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import posed, {PoseGroup} from 'react-pose';
```

Comment:
Is this component tree codesplit from the main bundle?

Reply:
I'll have it codesplit in the routes.

Reply:
Yeah it's codesplit here

https://github.com/getsentry/sentry/pull/13108/files#diff-2153e45c44c16949b38b6aa2af25d1eaR758